### DISCUSSION TITLE
Phone instead of Pi

### DISCUSSION THREAD

**Discussion by bstmedia** [tesla-android/issue-tracker#61] [General]
Hi Folks, 

first of all. Thank you all for the great work.  I was browsing the documentation and started to wonder if i could theoretical connect a phone directly to the HDMI to CSI-2 adapter and skip the carlinkit and second PI. Its not wireless. But would cut cost in half and simplify the setup. 

Would that work? Is there any other disadvantage?

Regards
Stefan

**Comment by DORNINEM** [comment]
No Carlinkit... No Apple Carplay nor Android Auto 😜

**Reply by bstmedia** [reply]
Thanks for the reply.  I was reading more and found the following issue.
https://github.com/tesla-android/issue-tracker/discussions/7

The resultion might cause an issue too. I will stick with hardware list suggested and give it a try.
